Lib/idlelib/pyshell.py

```diff
@@ -902,10 +902,11 @@ def __init__(self, flist=None):
         self.save_stderr = sys.stderr
         self.save_stdin = sys.stdin
         from idlelib import iomenu
-        self.stdin = PseudoInputFile(self, "stdin", iomenu.encoding)
-        self.stdout = PseudoOutputFile(self, "stdout", iomenu.encoding)
-        self.stderr = PseudoOutputFile(self, "stderr", iomenu.encoding)
-        self.console = PseudoOutputFile(self, "console", iomenu.encoding)
+        self.stdin = StdInFile(self, "stdin", iomenu.encoding)
+        self.stdout = StdOutFile(self, "stdout", iomenu.encoding)
+        self.stderr = StdOutFile(self, "stderr", iomenu.encoding,
+                                 "backslashreplace")
```

Comment:
In the Windows REPL,
```
>>> sys.stdin.errors
'surrogateescape'
>>> sys.stdout.errors
'surrogateescape'
>>> sys.stderr.errors
'backslashreplace'
```
If we should match the 3rd, 'backslashreplace', why not the first two and use 'surrogateescape'?